js/scorm/wrapper.js

```diff
@@ -581,45 +581,41 @@ define (function(require) {
 
 	/**
 	* Converts milliseconds into the SCORM 2004 data type 'timeinterval (second, 10,2)'
-	* this will output something like 'PT2H5M10S' a value which indicates a period of time of 2 hours, 5 minutes & 10 seconds
+	* this will output something like 'P1DT3H5M0S' which indicates a period of time of 1 day, 3 hours and 5 minutes
+	* or 'PT2M10.1S' which indicates a period of time of 2 minutes and 10.1 seconds
 	*/
 	ScormWrapper.prototype.convertToSCORM2004Time = function(msConvert) {
-
-		var timeinterval = "";
 		var csConvert = Math.floor(msConvert / 10)
-
+  
```

Comment:
not so much the whitespace (which was there before) but the missing semi-colon (which was also there before but I will fix)